catalog/spatial/registry/registry-admin-modules/registry-admin-poller-service/src/main/java/org/codice/ddf/registry/admin/poller/AdminPollerServiceBean.java

```diff
@@ -201,8 +215,77 @@ public boolean sourceStatus(String servicePID) {
 
     @Override
     public List<Map<String, Object>> allSourceInfo() {
+        List<Map<String, Object>> allSourceInfo = new ArrayList<>();
+        List<Map<String, Object>> registryMetacardInfo = new ArrayList<>();
+
+        List<Map<String, Object>> sourceInfo = allMetatypeInfo(SOURCE_FILTER);
+
+        try {
+            List<RegistryPackageType> registryMetacardObjects =
+                    federationAdminService.getRegistryObjects();
+            for (RegistryPackageType metacardObject : registryMetacardObjects) {
+                registryMetacardInfo.add(RegistryPackageWebConverter.getRegistryObjectWebMap(
+                        metacardObject));
+            }
+        } catch (FederationAdminException e) {
```

Comment:
Good question. I'm not totally sure if or what we should do more than the logging message, but if you don't get any registry metacards you can still access the source metatype info this way. So I think we want to keep going...